src/Data/Nat/Binary/Properties.agda

```diff
@@ -730,6 +740,15 @@ private
   { isCommutativeMonoid = +-0-isCommutativeMonoid
   }
 
+import Algebra.Properties.CommutativeSemigroup +-commutativeSemigroup
+  as Bin+CSemigroup
+import Algebra.Morphism.TwoMagmas +-magma ℕₚ.+-magma
+  as Magmas+ℕᵇ-ℕ
```

Comment:
`import` statements should be at the top of the file

Reply:
But they need the parameter values  +-commutativeSemigroup, +-magma               
(for `ℕᵇ`) which are implemented not at the top of this file but only several lines higher.                                                             
- ?